server/src/interfaces/join-response.interface.ts

```diff
@@ -2,6 +2,7 @@ import { Response } from './response.interface';
 import { Player } from './player.interface';
 
 export interface JoinResponse extends Response {
+  connected: Array<JudgeRole>;
```

Comment:
To jest oczywiście w porządku, chciałem tylko zwrócić uwagę, że według [zasad nazewnictwa TypeScript](https://google.github.io/styleguide/tsguide.html#arrayt-type), dla takich prostych typów preferowany jest zapis `T[]` zamiast `Array<T>` i tak też ogólnie piszemy w kodzie na backendzie.